src/.vuepress/theme/Search.vue

```diff
@@ -62,8 +61,19 @@ export default {
         docsearch = docsearch.default;
         docsearch({
           inputSelector: '#algolia-search-input',
-          apiKey: 'c6452010dc26eb671d637214f1110c91',
-          indexName: 'kuzzle',
+
+          // KUZZLEIO APP
+          apiKey: 'de63216cd8d0116b2755916b9a38ae35',
+          indexName: 'docs',
+          appId: 'VF5HP4ZVDU',
+
+          // DOCSEARCH APP
+          // apiKey: 'c6452010dc26eb671d637214f1110c91',
```

Comment:
forget to delete

Reply:
I keep this because very soon we will switch to the DOCSEARCH application. Our private index will be then used for the next-docs instance.